Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+
+    <PropertyGroup>
+        <LangVersion>latest</LangVersion>

```

Comment:
One project was giving me problems here, so I just added this to the root

Reply:
We manage such things on the solution level from https://github.com/Lombiq/.NET-Analyzers/blob/dev/Build.props#L12 so let's not add this here. However, since we specify a concrete version there, it should be safe to use `latest` in these repos but not copied everywhere like this. It can be added from the build with `-p` though.

Same for HE.

Reply:
I'll move it to -p.

Which reminds me, which dotnet version do you want to use for publishing? that would also resolve it (even though you're building for 3, using 6 as a build tool would possibly resolve this)

Reply:
Having it explicitly defined looks better though because then the publishing target and the SDK used to do the publish can differ.

Do you mean which SDK to use for build on the agent? Currently, we use .NET 5 everywhere since 6 causes some issues. Is this something we can manage centrally too?

Reply:
Yes I’ll set net 5 in the pipelines. So it’s explicitly set there in one place

Reply:
Great then.

Reply:
Done to .NET 5